python_utils/stack-pt-merge.py

```diff
@@ -533,56 +481,54 @@ def iter_days():
 
     # Create output file
     print_heading("Creating output file ...")
-    c = nc.Dataset(ofn, "w")
-    point_dim = c.createDimension(POINT_DIM_NAME, npoint_tot)
+    ds_out = nc.Dataset(ofn, "w")
+    _ = ds_out.createDimension(POINT_DIM_NAME, npoint_tot)
     ntime = nstep
     tstep = 1  # hourly
 
     if not stack_groups_only:
-        time_dim = c.createDimension("time", None) # unlimited for ntime
-        time_coord = c.createVariable("TIME", np.int32, ("time",))
+        _ = ds_out.createDimension("time", None)  # unlimited for ntime
```

Comment:
@ytangnoaa was there an issue with this being `ntime` instead of `None` (unlimited)?

Reply:
> @ytangnoaa was there an issue with this being `ntime` instead of `None` (unlimited)?

Raffaele requested the time step must be "unlimited" so his code could read it correctly.